src/app/views/ChatEnable.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+  import {WRAP} from "@welshman/util"
+  import {repository} from "@welshman/app"
+  import {canDecrypt, ensureUnwrapped, env} from "src/engine"
+  import {router} from "src/app/util"
+  import Anchor from "src/partials/Anchor.svelte"
+
+  let loading = false
+
+  const enableChat = async () => {
+    for (const event of repository.query([{kinds: [WRAP]}])) {
+      ensureUnwrapped(event)
+    }
+
+    canDecrypt.set(true)
+  }
+  const submit = async () => {
+    loading = true
+
+    try {
+      await enableChat()
+    } finally {
+      loading = false
+      router.at("channels").push()
```

Comment:
This adds a history item, when we probably want to remove one instead. I think `router.pop()` will be better here.